Dockerfile

```diff
@@ -7,18 +7,15 @@ WORKDIR /tmp
 # install basic packages
 #-----------------------
```

Comment:
The image used in this Dockerfile is  `centos:7.6.1810`
It must be something more recent to match OSP 16 version which is RHEL 8.X based. Also, since centos is disconinued in favor of CentOS Stream which would not be acceptable as base image consider using UBI 8.4 as base image, more specifically  ` registry.access.redhat.com/ubi8/ubi:8.4-203.1622660121`

Reply:
https://www.centos.org/centos-stream/

Reply:
This being only a build container it does not matter much but still would be good to not fall too much behind. All in all registry.access.redhat.com/ubi8/ubi:8.4-203.1622660121 seems like a good image to be up-to-date

Reply:
also, in .travis.yaml increment version to 3.0.0 since this is a major change with OS and python 2 to 3 update\

```
- VERSION=2.0.0
``